pyproject.toml

```diff
@@ -52,6 +53,7 @@ module = [
   "plotly.*",
   "pydash.*",
   "pymongo.*",
+  "RestrictedPython.*",
```

Comment:
The RestrictedPython and AccessControl modules aren't types so they need to be added to the exception list